test/test_poolmanager.py

```diff
@@ -128,6 +129,29 @@ def test_http_pool_key_fields(self):
         )
         assert all(isinstance(key, PoolKey) for key in p.pools.keys())
 
+    @pytest.mark.skipif(sys.version_info < (3, 7), reason="requires python 3.7+")
+    def test_http_pool_key_blocksize(self):
+        """Assert blocksize is honored when selecting a pool."""
+        connection_pool_kw = {"blocksize": 16384}
+        p = PoolManager()
+        conn_pools = [
+            p.connection_from_url("http://example.com/"),
+            p.connection_from_url("http://example.com:8000/"),
+            p.connection_from_url("http://other.example.com/"),
+        ]
+
+        for key, value in connection_pool_kw.items():
+            p.connection_pool_kw[key] = value
+            conn_pools.append(p.connection_from_url("http://example.com/"))
```

Comment:
We don't need this loop because we just have one key in `connection_pool_kw`. SO:

```suggestion
        p.connection_pool_kw["blocksize"] = 16384
        conn_pools.append(p.connection_from_url("http://example.com/"))
```

Also, we can remove variable `connection_pool_kw` at the beginning of test